stigs/system/ksp-block-stig-rhel-v-230336.yaml

```diff
@@ -0,0 +1,22 @@
+# KubeArmor is an open source software that enables you to protect your cloud workload at run-time.
+# To learn more about KubeArmor visit:
+# https://www.accuknox.com/kubearmor/
+
+apiVersion: security.kubearmor.com/v1
+kind: KubeArmorPolicy
+metadata:
+  name: ksp-block-stig-rhel-v-230336
```

Comment:
@nandakrr the file name is wrong. since this is an audit policy the file name should start with `ksp-audit-`
cc: @salman-accuknox @syeddhadi @raviknox